palindrom-polymer.html

```diff
@@ -0,0 +1,219 @@
+<!-- palindrom-polymer-client version: 4.0.0 | MIT License -->
+
+<link rel="import" href="../polymer/polymer.html">
+<!-- <link rel="import" href="iron-request.html"> -->
+
+<!-- include Palindrom with dependencies -->
+<script src="../Palindrom/dist/palindrom-dom.min.js"></script>
+
+<!--
+The Polymer version of `palindrom-polymer` element binds [Palindrom](https://github.com/Palindrom/Palindrom) with [Polymer's template binding](https://www.polymer-project.org/1.0/docs/devguide/templates.html).
+That keeps your Polymer app, or just `dom-bind` template in sync with any server-side
+data-model using Palindrom & [JSON Patch](https://tools.ietf.org/html/rfc6902) flow.
+
+You get three-way data binding server - JS - HTML, kept in flawless sync.
+
+    <palindrom-polymer
+        obj="{{model}}"></palindrom-polymer>
+
+
+It establishes the Palindrom connection when attached. All the changes made
+in browser are sent to the server via WebSocket or HTTP, as
+[JSON Patch](https://tools.ietf.org/html/rfc6902)es.
+All the changes from server are also received and propagated to your HTML.
+
+@demo demo/index.html
+-->
+<script>
+    var component = (function () {
+        'use strict';
+
+        var component = {
+
+            is: 'palindrom-polymer',
+
+            properties: {
+                obj: {
+                    type: Object,
+                    notify: true,
+                    value: function () {
+                        return {};
+                    }
+                },
+                target: {
+                    type: String,
+                    notify: true,
+                    value: 'palindrom-connection'
+                }
+            },
+            attached: function () {
+                const palindromConnection = document.querySelector(this.target);
```

Comment:
Apparently, `getRootNode` only works in Safari 10.1+, and SauceLabs has Safari 10 only. Do you have a walkaround?